### DISCUSSION TITLE
【测试】切分法发大图片

### DISCUSSION THREAD

**Discussion by capacitor1** [share121/inter-knot#1364] [其他]
《论分卷》
原图96MB，切分后总大小132MB
![1](https://github.com/user-attachments/assets/15c63733-fb27-4769-924d-8ca3fc1cdf47)
![2](https://github.com/user-attachments/assets/ccfd6c63-c838-417a-bc3b-df7e1ed547c0)
![3](https://github.com/user-attachments/assets/8fcfc101-2780-4f9f-81ce-955ecb23ac0f)
![4](https://github.com/user-attachments/assets/587b41fa-7863-4398-921f-859ae1ea078d)
![5](https://github.com/user-attachments/assets/fa22be90-5ba8-444a-8614-28ac06bf1fc8)
![6](https://github.com/user-attachments/assets/b67b1f65-c5dd-468b-8803-2d5c6e470dc3)
![7](https://github.com/user-attachments/assets/42b92234-ca06-42be-a8e1-33c1ae5a37c6)
![8](https://github.com/user-attachments/assets/ef1f1004-2926-4c00-9ea0-19b5b3ec4a48)
![9](https://github.com/user-attachments/assets/03ae1093-d6fa-46d0-a3f2-d739f47ce086)
![10](https://github.com/user-attachments/assets/fcbe287c-538e-481e-89b6-dc0d01734678)
![11](https://github.com/user-attachments/assets/3ab1052f-1a3c-4a2a-a8bb-157165fc3c99)
![12](https://github.com/user-attachments/assets/13d25d41-30c2-4eb6-8bd1-30df0fb12fad)
![13](https://github.com/user-attachments/assets/31aedea0-5022-4938-8213-ec2ce51f3d9a)
![14](https://github.com/user-attachments/assets/9cc2ee6f-cb2e-48d9-ab23-4955eebc1d26)
![15](https://github.com/user-attachments/assets/e4ba3747-1da7-44bc-8bba-ba36553839da)
![16](https://github.com/user-attachments/assets/edd16b7a-9b7d-4998-9d89-887c9a3a892b)
![17](https://github.com/user-attachments/assets/4d6fef91-23f5-4201-89dd-a1dc0addc5d4)
![18](https://github.com/user-attachments/assets/fc104642-e746-443b-9c91-0b2221efbb41)
![19](https://github.com/user-attachments/assets/47379588-05e9-4c70-a3ea-a283bd48039e)
![20](https://github.com/user-attachments/assets/613d6c26-28d2-47bf-aa4f-eed462663efa)

**Comment by capacitor1** [comment]
中间有白边。。这很难受

**Comment by Myitian** [comment]
我感觉 100M 的限制够用了
一般 10000x10000 的图比 100M 小的
就算完全不压缩都能存 5912x5912x3 = 104855232
一般也不会发几万x几万的图
